ch-covidcertificate-backend-delivery/ch-covidcertificate-backend-delivery-ws/src/main/java/ch/admin/bag/covidcertificate/backend/delivery/ws/config/BaseSecurity.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2020 Ubique Innovation AG <https://www.ubique.ch>
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+package ch.admin.bag.covidcertificate.backend.delivery.ws.config;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.annotation.Order;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+
+/**
+ * Inject this security config, to overrule the default spring boot security config which
+ * goes into complete lock down.
+ * 
+ * We further deactivate csrf, since we are not serving any HTML/Frontend content but only serve
+ * as an API-Server
+ */
+@Configuration
+
+@Order(10000)
+public class BaseSecurity extends WebSecurityConfigurerAdapter {
+    @Override
+    protected void configure(HttpSecurity http) throws Exception {
```

Comment:
Would probably also work, when we change the` @EnableAutoConfiguration` Annotation in [Application.java](https://github.com/admin-ch/CovidCertificate-App-Certificate-Delivery/blob/main/ch-covidcertificate-backend-delivery/ch-covidcertificate-backend-delivery-ws/src/main/java/ch/admin/bag/covidcertificate/backend/delivery/ws/Application.java)
to `@EnableAutoConfiguration(exclude=SecurityAutoConfiguration.class)`

But as it is now, it works aswell